cpp/README.md

```diff
@@ -43,12 +43,11 @@ If the compilation is successful, `*.tvc` file is created in the directory where
 
 ## Contract deployment
 
-Here we describe a contract deployment to the TON Blockchain Test Network (testnet) using `tonlabs-cli` tool.
-Note: for MacOS X replace `tonlabs-cli` with `cargo run` in the following instruction, and the run is supposed to be made from the directory where `tonlabs-cli` is built.
+Here we describe a contract deployment to the TON Blockchain Test Network (testnet) using `tonlabs-cli` tool. The following commands must be run from the directory where tonlabs-cli is located.
 
 1. Generate contract address
 ```
-tonlabs-cli genaddr Contract.tvc Contract.abi --genkey key
+cargo run Contract.tvc Contract.abi --genkey key
```

Comment:
Are you sure that you don't need `genaddr` or other command here?

Reply:
Fixed